pkg/cmd/runsPrepare.go

```diff
@@ -93,7 +93,10 @@ func executeAssemble(cmd *cobra.Command, args []string) {
 	}
 
 	// Create an API client
-	launcher := launcher.NewRemoteLauncher(bootstrapData.ApiServerURL)
+	launcher := launcher.NewRemoteLauncher(bootstrapData.ApiServerURL, fileSystem, galasaHome)
+	if err != nil {
```

Comment:
This code has a ton of 'err' variables being re-declared. I assume this is safe, but might be worth making them '=' instead of ':=' ?

Reply:
Addressed in https://github.com/galasa-dev/cli/pull/151/commits/f49da0d4680b9141d803a0d0668ab11e5d8b27ee